src/garage/tf/policies/continuous_mlp_policy.py

```diff
@@ -159,6 +179,10 @@ def clone(self, name):
 
         Args:
             name (str): Name of the newly created policy.
+
+        Returns:
+            garage.tf.policies.continuous_mlp_policy: Clone of this object
```

Comment:
this is a package, not a class

i think you meant `garage.tf.policies.ContinuousMLPPolicy`

Reply:
Ah, yes. I fixed it.